Sources/UIKit/NSLayoutConstraint.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  NSConstraintLayout.swift
+//  Alicerce
+//
+//  Created by Tiago Veloso on 15/02/2018.
+//  Copyright © 2018 Mindera. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension NSLayoutConstraint {
+
+
+    /// Similar to NSLayoutConstraint.activate(), but this method ensures that the views
+    /// that are adding the constraints disable translatesAutoresizingMaskIntoConstraints
+    ///
+    /// - Parameter constraints: an array of constrains to activate
+    public static func add(constraints: [NSLayoutConstraint]) {
```

Comment:
I think `add` is not clear enough as to what this API does, since we're essentially activating the constraints and introducing a side effect (marking the `translatesAutoresizingMaskIntoConstraints` flag as `false`).

I know this is more verbose, but perhaps:
`func activateAndDisableAutoresizingMasksTranslation(_ constraints: [NSLayoutConstraint])`?

Or, we could make a twist and make disabling the masks a parameter:
`func activate(_ constraints: [NSLayoutConstraint], disableAutoresizingMasksTranslation: Bool = true)`

What do you say?

Reply:
`activate` is the instance method, which for me, nhec 😕